lib/lockbox/calculations.rb

```diff
@@ -3,7 +3,10 @@ module Calculations
     def pluck(*column_names)
       return super unless model.respond_to?(:lockbox_attributes)
 
-      lockbox_columns = column_names.map.with_index { |c, i| [model.lockbox_attributes[c.to_sym], i] }.select(&:first)
+      lockbox_columns = column_names.map
+                                    .with_index { |c, i| [model.lockbox_attributes[c.to_sym], i] }
+                                    .select(&:first)
+                                    .reject { |la, _i| la.fetch(:migrating, false) }
```

Comment:
Let's keep on a single line and make it part of the `select`.

Reply:
Alright - good point!

I moved the extra filtering into the `select`. I tried to keep it "verbose" enough, so that it still makes sense for people looking at this in a few weeks down the road.

Let me know if that's what you had in mind.

Reply:
Looks perfect 👍